diff -ura --color dwm-6.1/config.h dwm-6.1.new/config.h
--- dwm-6.1/config.h	2018-08-08 01:07:27.951933319 +0200
+++ dwm-6.1.new/config.h	2018-08-08 01:09:54.958026659 +0200
@@ -2,15 +2,15 @@
 
 /* appearance */
 static const char *fonts[] = {
-	"monospace:size=10"
+	"xos4 Terminus:pixelsize=14:antialias=false:autohint=false"
 };
-static const char dmenufont[]       = "monospace:size=10";
-static const char normbordercolor[] = "#444444";
-static const char normbgcolor[]     = "#222222";
-static const char normfgcolor[]     = "#bbbbbb";
-static const char selbordercolor[]  = "#005577";
-static const char selbgcolor[]      = "#005577";
-static const char selfgcolor[]      = "#eeeeee";
+static const char dmenufont[]       = "xos4 Terminus:pixelsize=14:antialias=false:autohint=false";
+static const char normbordercolor[] = "#282828";
+static const char normbgcolor[]     = "#282828";
+static const char normfgcolor[]     = "#ebdbb2";
+static const char selbordercolor[]  = "#98971a";
+static const char selbgcolor[]      = "#98971a";
+static const char selfgcolor[]      = "#f9f5d7";
 static const unsigned int borderpx  = 1;        /* border pixel of windows */
 static const unsigned int snap      = 32;       /* snap pixel */
 static const int showbar            = 1;        /* 0 means no bar */
