diff -ura --color dwm-6.1/config.h dwm-6.1.new/config.h
--- dwm-6.1/config.h	2018-08-08 01:35:16.882091132 +0200
+++ dwm-6.1.new/config.h	2018-08-08 01:40:32.270861111 +0200
@@ -57,6 +57,10 @@
 static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", normbgcolor, "-nf", normfgcolor, "-sb", selbgcolor, "-sf", selfgcolor, NULL };
 static const char *termcmd[]  = { "kitty", NULL };
 
+static const char *upvol[]   = { "amixer", "set", "Master", "3+", NULL };
+static const char *downvol[] = { "amixer", "set", "Master", "3-", NULL };
+static const char *mutevol[] = { "amixer", "set", "Master", "toggle", NULL };
+
 static Key keys[] = {
 	/* modifier                     key        function        argument */
 	{ MODKEY,                       XK_p,      spawn,          {.v = dmenucmd } },
@@ -92,6 +96,9 @@
 	TAGKEYS(                        XK_8,                      7)
 	TAGKEYS(                        XK_9,                      8)
 	{ MODKEY|ShiftMask,             XK_q,      quit,           {0} },
+	{ MODKEY,												XK_F12,		spawn,						{.v = upvol } },
+	{ MODKEY,												XK_F11,		spawn,						{.v = downvol } },
+	{ MODKEY,												XK_F10,		spawn,						{.v = mutevol } },
 };
 
 /* button definitions */
Only in dwm-6.1: drw.o
Only in dwm-6.1: dwm
Only in dwm-6.1: dwm.o
Only in dwm-6.1.new: terminal.patch
Only in dwm-6.1: util.o
